src/sources/soundsourcesndfile.h

```diff
@@ -0,0 +1,20 @@
+#ifndef SOUNDSOURCESNDFILE_H
+#define SOUNDSOURCESNDFILE_H
+
+#include "sources/soundsource.h"
+
+class SoundSourceSndFile: public Mixxx::SoundSource {
+    typedef SoundSource Super;
```

Comment:
My typedef/alias "Super" is a simple indirection to have a common and way to refer to the singular base class, if there is such. Unfortunately C++ does not provide a concept to tag one of the multiple base classes as a "super" class.